homeassistant/components/todo/services.yaml

```diff
@@ -0,0 +1,84 @@
+create_item:
+  target:
+    entity:
+      domain: todo
+      supported_features:
+        - todo.TodoListEntityFeature.CREATE_TODO_ITEM
+  fields:
+    summary:
+      required: true
+      example: "Submit Income Tax Return"
+      selector:
+        text:
+    status:
+      required: false
+      example: "needs_action"
+      selector:
+        select:
+          translation_key: status
+          options:
+            - needs_action
+            - completed
+update_item:
+  target:
+    entity:
+      domain: todo
+      supported_features:
+        - todo.TodoListEntityFeature.UPDATE_TODO_ITEM
+  fields:
+    uid:
+      required: false
+      selector:
+        text:
+    summary:
+      required: false
+      example: "Submit Income Tax Return"
+      selector:
+        text:
+    status:
+      required: false
+      example: "needs_action"
+      selector:
+        select:
+          translation_key: status
+          options:
+            - needs_action
+            - completed
+delete_item:
+  target:
+    entity:
+      domain: todo
+      supported_features:
+        - todo.TodoListEntityFeature.DELETE_TODO_ITEM
+  fields:
+    uid:
+      required: false
+      selector:
+        object:
+    summary:
+      required: false
+      selector:
+        object:
+move_item:
+  target:
+    entity:
+      domain: todo
+      supported_features:
+        - todo.TodoListEntityFeature.MOVE_TODO_ITEM
+  fields:
+    uid:
+      required: true
```

Comment:
All these parameters are optional now.

Reply:
Corrected, then I went ahead and removed all the `required: false` given the default is already false.